ezcontent_publish.module

```diff
@@ -103,7 +103,9 @@ function ezcontent_publish_form_shortcut_set_customize_form_alter(&$form, $form_
  */
 function ezcontent_publish_form_alter(&$form, $form_state, $form_id) {
   $currentUser = \Drupal::currentUser();
-  $form['shortcuts']['links']['#empty'] = t('No shortcuts available. <a href=":link">Add a shortcut</a>', [':link' => Url::fromRoute('ezcontent_publish.shortcut.link_add', ['shortcut_set' => 'shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id()])->toString()]);
+  if ($currentUser->id() != '1' ) {
```

Comment:
if $currentUser is being used only here, then let's avoid it and use \Drupal::currentUser()->id() directly.